integration/tests/basic/test_rpc_calls.py

```diff
@@ -101,6 +103,13 @@ def get_event_signatures(abi: tp.List[tp.Dict]) -> tp.List[str]:
 class TestRpcCalls(BaseMixin):
     _erc20_contract: tp.Optional[tp.Any] = None
 
+    @pytest.fixture(scope="class")
+    def event_caller_contract(self, web3_client, class_account) -> typing.Any:
```

Comment:
Let's move it to conftest and delete duplication (for example test_event_logs.py )

Reply:
Good point, done!